docs/index.rst

```diff
@@ -67,26 +67,26 @@ Example Usage
 
 ::
 
+    from locasticsearch import Locasticsearch
     from datetime import datetime
-    from elasticsearch import Elasticsearch
-    es = Elasticsearch()
+
+    es = Locasticsearch()
```

Comment:
What is locasticsearch? Typo?

Reply:
it is a new POC library I am experimenting with. But definitely out of place here :-)